examples/core/src/main/scala/com/twitter/util/Credentials.scala

```diff
@@ -36,7 +36,7 @@ object Credentials {
     private[this] val key: _root_.scala.util.matching.Regex = "[\\w-]+".r
     private[this] val value: _root_.scala.util.matching.Regex = ".+".r
 
-    def auth: Parser[(_root_.scala.Predef.String, _root_.scala.Predef.String)] = key ~ ":" ~ value ^^ { case k ~ ":" ~ v => (k, v) }
+    def auth: parser.this.Parser[_root_.scala.Tuple2[_root_.scala.Predef.String, _root_.scala.Predef.String]] = key ~ ":" ~ value ^^ { case k ~ ":" ~ v => (k, v) }
```

Comment:
I'm not intuitively understanding what items need _root_, like why scala.Tuple2 needs it but parser.this.Parser does not.

Reply:
Current version of RscCompat qualifies everything as much as possible. In the case of `this` types, their qualifier must be an identifier, so we have to say `parsers.this.Parser`, not `com.twitter.util.Credentials.parser.this.Parser`.